spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/TaskJobService.java

```diff
@@ -155,13 +155,15 @@
 	 * Retrieves Pageable list of {@link JobExecutionWithStepCount}s from the JobRepository
 	 * and matches the data with a task id but excludes the step executions.
 	 *
+	 *
+	 * @param queryString search query string to filter job names
 	 * @param pageable enumerates the data to be returned.
 	 * @return List containing {@link TaskJobExecution}s.
 	 *
 	 * @throws NoSuchJobExecutionException thrown if the job execution specified does not
 	 *     exist.
 	 */
-	List<TaskJobExecution> listJobExecutionsWithStepCount(Pageable pageable) throws NoSuchJobExecutionException;
```

Comment:
Since we are at a minor version, we don't want to change an existing API. Meaning lets keep the original definition as an overloaded call to the listJobExecutionsWithStepCount(String queryString, Pageable pageable), in that it passes a `null` for the query string.

Reply:
Oh, okay. But then how can we have 2 APIs exposed to the same GET Mapping "" with different params?